### PR TITLE
Events: sender_type property to differentiate between multiple event sources

### PR DESCRIPTION
**Pull request by martastain**
Endpoints that accepted "x-sender" header now also accept "x-sender-type". 
sender_type is stored in the database in the separate field. and it is available in the eventmodel.

@jakubjezek001 @iLLiCiTiT do you need to be able to filter by it in enroll somehow? if so, how?

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
All works good, but I have one question, enrolled events don't have sender type filled from headers and enroll endpoint does not allow to fill it.

**Comment by martastain** [comment]
> All works good, but I have one note, enrolled events don't have sender type filled from headers and enroll endpoint does not allow to fill it.

Thanks. Good catch. I've added senderType to the request body and the enroll endpoint also supports providing sender and sender type values in headers. Both is optional, but when a value is specified both in the headers and the body, value from the body is used.

File: schemas/schema.public.update.sql [hunk 1]
```diff
@@ -467,3 +467,24 @@ CREATE OR REPLACE FUNCTION add_meta_column_to_thumbnails()
 SELECT add_meta_column_to_thumbnails();
 DROP FUNCTION IF EXISTS add_meta_column_to_thumbnails();
 
+
+----------------
+-- AYON 1.5.3 --
```
[the remark this answers is not in the document]
Comment (martastain) [reply schemas/schema.public.update.sql:472]: I'll change it before merging. it will be either 1.5.6 or 1.6.0

**Comment by iLLiCiTiT** [comment]
> Thanks. Good catch. I've added senderType to the request body and the enroll endpoint also supports providing sender and sender type values in headers.

Nope, you didn't 🙂 , enroll still without `sourceType`.

**Comment by martastain** [comment]
@iLLiCiTiT i ALMOST did. it was just one `git commit` and  `git push` away :) sorry

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**